src/common/output.rs

```diff
@@ -13,107 +12,90 @@ pub enum Mode {
 const BACKUP_PREFIX: &str = "Original file was backed up to: ";
 
 /// Wraps warnings to provide more information when printing.
+#[derive(Clone, Copy)]
 pub struct Output {
```

Comment:
I think we should split this module to several: `Check` & `Fix`.
In the future we will add more commands like `compare` and for these commands we will have to create separate outputs.

Reply:
I like this idea 👍
I'll come back to this soon.

Reply:
> I think we should split this module to several: `Check` & `Fix`.

In fact, the method of separation was not so obvious.
The calling code for `Fix` and `Check` is strongly interconnected, which I think cannot be said about `Compare`.
So I had to add some conditions/logic to `dotenv_linter::run`, but I tried to make it the most obvious and concise way.

In general, I considered various options: traits, general output configuration structure, etc. But it looks a bit cumbersome for simple output functions and somehow framed `Fix` and `Check` into something combined, complicating understanding in the case of adding a `compare`.

Thus, we have split modules with their own methods. Please take a look at this 👀 
I will be glad to hear any suggestions and other options 🙂